docs/tr/docs/python-types.md

```diff
@@ -32,88 +32,88 @@ John Doe
 ```
 
-### Düzenle
+### Düzenleme
 
-Bu çok basit bir program.
+Bu program son derece basitti.
 
-Ama şimdi sıfırdan yazdığınızı hayal edin.
+Şimdi, bu programı sıfırdan yazmaya başladığınızı düşünün.
 
-Bir noktada fonksiyonun tanımına başlayacaktınız, parametreleri hazır hale getirdiniz...
+İlk adım, fonksiyonun tanımına başlamak ve gerekli parametreleri hazırlamaktır.
 
-Ama sonra "ilk harfi büyük harfe dönüştüren yöntemi" çağırmanız gerekir.
+Ancak metin içerisindeki ilk harfi büyük harfe dönüştüren metodu çağırmanız gerekecek.
 
- `upper` mıydı ? Yoksa  `uppercase`' mi? `first_uppercase`? `capitalize`?
+`upper` mıydı? Yoksa  `uppercase`' mi? `first_uppercase`? Ya da `capitalize`?
 
-Ardından, programcıların en iyi arkadaşı olan otomatik tamamlama ile denediniz.
+Ardından, programcıların en iyi dostu olan otomatik tamamlamayı kullanarak işinizi kolaylaştırdınız.
 
-'first_name', ardından bir nokta ('.') yazıp otomatik tamamlamayı tetiklemek için 'Ctrl+Space' tuşlarına bastınız.
+`first_name`, ardından bir nokta ('.') yazıp otomatik tamamlamayı tetiklemek için 'Ctrl+Space' tuşlarına bastınız.
 
 Ancak, ne yazık ki, yararlı hiçbir şey elde edemediniz:
 
 <img src="/img/python-types/image01.png">
 
-### Tipleri ekle
+### Tipleri Ekle
 
 Önceki sürümden sadece bir satırı değiştirelim.
 
-Tam olarak bu parçayı, işlevin parametrelerini  değiştireceğiz:
+Tam olarak bu parçayı, işlevin parametrelerini şu şekilde değiştireceğiz:
 
 ```Python
     first_name, last_name
 ```
 
-ve bu hale getireceğiz:
+ve bu hale getirmiş olacağız:
 
 ```Python
     first_name: str, last_name: str
 ```
 
-Bu kadar.
+İşte bu kadar!
 
-İşte bunlar "tip belirteçleri":
+"Tip belirteçlerini" aşağıdaki gibi uygulayabilirsiniz:
 
 ```Python hl_lines="1"
 {!../../../docs_src/python_types/tutorial002.py!}
 ```
 
-Bu, aşağıdaki gibi varsayılan değerleri bildirmekle aynı şey değildir:
+Parametrelere varsayılan değer atamak ile aynı şey değildir:
 
 ```Python
     first_name="john", last_name="doe"
 ```
 
-Bu tamamen farklı birşey
+Dolayısıyla bu iki kullanımın tamamen birbirinden farklı.
 
-İki nokta üst üste (`:`) kullanıyoruz , eşittir (`=`) değil.
+Dikkat edin; iki nokta üst üste (`:`) kullanıyoruz , eşittir (`=`) DEĞİL.
 
-Normalde tip belirteçleri eklemek, kod üzerinde olacakları değiştirmez.
+Ayrıca tip ipuçları eklemek kodun çıktısını değiştirmez.
 
-Şimdi programı sıfırdan birdaha yazdığınızı hayal edin.
+Şimdi programı tekrar en baştan yazdığınızı hayal edin.
 
-Aynı noktada, `Ctrl+Space` ile otomatik tamamlamayı tetiklediniz  ve şunu görüyorsunuz:
+Aynı noktada, `Ctrl+Space` ile otomatik tamamlamayı tetiklediniz ve şunu görüyorsunuz:
 
 <img src="/img/python-types/image02.png">
 
 Aradığınızı bulana kadar seçenekleri kaydırabilirsiniz:
 
 <img src="/img/python-types/image03.png">
 
-## Daha fazla motivasyon
+## Daha Fazla Motivasyon
 
-Bu fonksiyon, zaten tür belirteçlerine sahip:
+Tür belirteçlerine sahip bir fonksiyonu inceleyelim:
```

Comment:
Dokümanın farklı yerlerinde `type hint` için hem `tür belirteçleri` hemde `tip belirteçleri` kullanılmış. Bu okuyucunun kafasını karıştırabilir. `Tip Belirteçleri` şeklinde sabit bir kullanıma yönelebilir miyiz? `type annotation` içinde `tip açıklamaları` kullanabiliriz.